usaspending_api/references/v1/views.py

```diff
@@ -10,15 +10,12 @@
 from django.utils.decorators import method_decorator
 
 
-@method_decorator(deprecated, name="post")
```

Comment:
This will have the side effect that users who use the v1 endpoint will not get a deprecation warning. Are we comfortable with that?

Reply:
I thought about that and think it shouldn't be a big deal considering the v1 endpoint references can be quickly removed. The front end impact is minimal and I submitted a PR for the required changes. Another solution would be creating a new file for the V2 code.

Reply:
Now that we're duplicating to v2, we should re-deprecate all this v1 stuff.